AAI/AAIConnectProfile.md

```diff
@@ -346,6 +342,8 @@ Only the GA4GH claims truly must be as proscribed here. Refer to OIDC Spec for m
  \<ga4gh-spec-claims\>
 }
 ```
+-   `<ga4gh-spec-claims>`: Claims included as part of a GA4GH standard specification based on the scopes provided. This content MAY be incomplete (i.e. a subset of data elements) and more may be fetched as indicated by ga4gh_userinfo_claims. A non-normative example of `<ga4gh-spec-claims>` is: "ga4gh": {[ga4gh claims](https://docs.google.com/document/d/11Wg-uL75ypU5eNu2p_xh9gspmbGtmLzmdq5VfPHBirE)}
```

Comment:
Update RI claims spec links like this to be: https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/RI_Claims_V1.md

Reply:
Can you do a PR to this branch? I'll merge.

Reply:
done